scss/components/_emergency-banner.scss

```diff
@@ -1,176 +1,135 @@
+// Colours are from gov.uk and not our usual design system choices
+$gov_black: #0b0c0c;
+$gov_red: #b10e1e;
+$gov_teal: #28a197;
+
+.page-type--homepage .emergency-banner {
+  padding: 27px 0;
+}
```

Comment:
Just checking that this is what we want. Doesn't this mean that the header is bigger in the homepage than in any other page?

Reply:
Yep

Reply:
![image](https://user-images.githubusercontent.com/19624419/147077900-529b5f09-d61d-497f-afeb-99f76408be2d.png)

Reply:
Thanks, yes I noticed that and only wondered if it was intentional 👍